src/lib.rs

```diff
@@ -356,26 +367,34 @@ pub trait Filesystem {
     fn mknod(
         &mut self,
         _req: &Request<'_>,
-        _parent: u64,
-        _name: &OsStr,
-        _mode: u32,
-        _umask: u32,
-        _rdev: u32,
+        parent: u64,
+        name: &OsStr,
+        mode: u32,
+        umask: u32,
+        rdev: u32,
         reply: ReplyEntry,
     ) {
+        debug!(
+            "[Not Implemented] mknod(parent: {:#x?}, name: {:?}, mode: {}, umask: {:#x?}, rdev: {})",
+            parent, name, mode, umask, rdev
+        );
         reply.error(ENOSYS);
     }
 
     /// Create a directory.
     fn mkdir(
         &mut self,
         _req: &Request<'_>,
-        _parent: u64,
-        _name: &OsStr,
-        _mode: u32,
-        _umask: u32,
+        parent: u64,
+        name: &OsStr,
+        mode: u32,
+        umask: u32,
         reply: ReplyEntry,
     ) {
+        debug!(
+            "[Not Implemented] mkdir(parent: {:#x?}, name: {:?}, mode: {}, umask: {:#x?})",
+            parent, name, mode, umask
+        );
         reply.error(ENOSYS);
     }
 
```

Comment:
Why not debug! on unlink?

Reply:
because I am unfortunately a fallible human being!